app/routers/profile.py

```diff
@@ -145,6 +147,6 @@ async def process_image(image, user):
 def get_image_crop_area(width, height):
     if width > height:
         delta = (width - height) // 2
-        return (delta, 0, width - delta, height)
```

Comment:
why?

Reply:
the sograim are marked redundant, but has no effect on code